app/pages/admin/project-status/experimental-features.jsx

```diff
@@ -25,7 +25,8 @@ const experimentalFeatures = [
   'general feedback',
   'slider',
   'highlighter',
-  'translator-role'
+  'translator-role',
+  'hide completed workflows'
```

Comment:
I'm inclined to make this the default behaviour for all projects, and not hide it behind an experimental feature flag.